src/lib/__tests__/InstantSearch-test.tsx

```diff
@@ -417,7 +417,7 @@ See https://www.algolia.com/doc/api-reference/widgets/configure/js/`);
 
     search.start();
 
-    await runAllMicroTasks();
+    await Promise.resolve();
```

Comment:
What if we use `wait(0)` here too?
Or does it make sense to bring `runAllMicroTasks` back and replace its implementation with `wait()`?

Reply:
we definitely can use `wait(0)`, but I'm not sure if it really means the same, this code happens before a timeout runs

Reply:
While I understand what you mean, I fear this subtleness that might not make any real life difference could complicate writing tests.

Reply:
makes sense, I'll check whether those tests can be done with wait too